root/static/scripts/edit/utility/copyFieldData.js

```diff
@@ -32,3 +32,34 @@ export default function copyFieldData<T>(
   targetField.value = sourceField.value;
   copyFieldErrors(sourceField, targetField);
 }
+
+
+export function copyPartialDateField(
+  sourceField: PartialDateFieldT,
+  targetField: WritablePartialDateFieldT,
+) {
```

Comment:
Usually Flow requires an annotated return type for exported functions (not sure why it doesn't complain here -- maybe it can infer void ones).  But better to be explicit anyway.
```suggestion
export function copyPartialDateField(
  sourceField: PartialDateFieldT,
  targetField: WritablePartialDateFieldT,
): void {
```